schema-loader/src/main/java/com/scalar/db/schemaloader/command/SchemaLoaderCommand.java

```diff
@@ -108,7 +108,7 @@ public Integer call() throws Exception {
     } else if (mode.deleteTables) {
       SchemaLoader.unload(configPath, schemaFile, coordinator);
     } else if (mode.repairTables) {
```

Comment:
It might be better to rename the field `repairTables` to `repairAll`.

```suggestion
    } else if (mode.repairAll) {
```

Reply:
Good catch, thank you.
https://github.com/scalar-labs/scalardb/pull/1203/commits/72b9136533e7a9519c4f2a0927937605354ac153